### ISSUE TITLE
weird checkbox toggling when selecting collections

### ISSUE THREAD

**Issue by raffaelj** [zontarian/hugocockpit#6]
I tested cockpit, hugo and your plugin the last days and it could be a nice replacement for jekyll with a great, self-hosted backend. The generation of my collections works well (on Windows7, Xampp with PHP7). I didn't get the run command to work, but maybe I'll figure it out later.

The weird issue is: I have multiple collections and select the ones for generating md files. The first time I click a checkbox, all collections get selected, the next time all get deselected and so on (in Firefox). In Opera and Chrome the checkboxes are always unselected.
This behavior is only visual. The script behind stores the information, which ones I selected and the console output says true/false after every click.

I tested it with the newest versions of
https://github.com/agentejo/cockpit/archive/master.zip and 
https://github.com/agentejo/cockpit/archive/next.zip

**Comment by zontarian** [comment]
Yes.. it's a problem with every browser that was not present a while ago. It's due to something in Riot.js.
I've corrected this in branch `feature/6-checkbox-toggling`, you can try it before I merge it to master.
Thanks for pointing out.

**Comment by raffaelj** [comment]
It works now. In master and in next branch.
